makelib/golang.mk

```diff
@@ -116,10 +116,12 @@ ifeq ($(GOOS),windows)
 GO_OUT_EXT := .exe
 endif
 
-# Detect when we're running in CI and output checkstyle XML at lint time rather
-# than human readable output.
 ifeq ($(RUNNING_IN_CI),true)
+# Output checkstyle XML rather than human readable output.
 GO_LINT_ARGS := --out-format=checkstyle > $(GO_LINT_OUTPUT)/checkstyle.xml
+
+# Output verbose tests that can be parsed into JUnit XML.
+GO_TEST_FLAGS := -v
```

Comment:
should we use `+=` here instead? e.g. `GO_TEST_FLAGS += -v`.  I see places in the build submodule doing that like https://github.com/upbound/build/blob/master/makelib/golang.mk#L74

Reply:
Probably - fixed.